_includes/manuals/1.9/3.2.2_installer_options.md

```diff
@@ -212,10 +212,6 @@ More information about compute resources can be found in the [Compute Resources
         <td>If disabled the EPEL repo will not be configured on RedHat family systems.</td>
       </tr>
       <tr>
-        <td style='white-space:nowrap'>--foreman-configure-ipa-repo</td>
```

Comment:
This table is auto-generated, so it needs to be removed from https://github.com/theforeman/puppet-foreman first.